packages/block-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@availity/block-ui",
```

Comment:
I know there is the `@availity` scope and all, but this will essentially publish a new/separate package. It's less of a migration and more of a forking at this point.

Reply:
That is my intention. Do you think I should update the existing repo instead? 

We have build/publish steps, Storybook, and other resources setup in `availity-react` that I think will make development easier if this is grouped with them. I also did not want to keep supporting the  `redux` component which I felt was easier to drop if I moved the code

Reply:
You can remove redux and all with a major version bump, that is fine. Everything can be achieved that you want, I would just suggest not changing the package name so it doesn't cause both to exist. Or add (via npm deprecate or w/e) that `react-block-ui` was replaced with `@availity/block-ui`.

Reply:
I will add a note to the other package that people can use this "new" package